orttraining/orttraining/core/session/training_session.cc

```diff
@@ -262,7 +262,7 @@ Status TrainingSession::ConfigureForTraining(
   }
 
   ORT_RETURN_IF_ERROR(ApplyTransformationsToMainGraph(trainable_initializers, config.graph_transformer_config,
-                                                      config_result));
+                                                      config_result, partition_by_row_));
```

Comment:
can we add this field `partition_by_row_` to the config_result instead of passing it explicitly?

Reply:
Added `partition_by_row` to `config_result`